### PR TITLE
[TypeInfo] Add tests for uncovered conditions

### PR DESCRIPTION
**Pull request by alexandre-daubois**
| Q             | A
| ------------- | ---
| Branch?       | 7.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | -
| License       | MIT

### CODE REVIEW

**Review by mtarld — APPROVED** [review]

File: src/Symfony/Component/TypeInfo/Tests/TypeResolver/ReflectionParameterTypeResolverTest.php [hunk 1]
```diff
@@ -44,6 +44,17 @@ public function testCannotResolveReflectionParameterWithoutType()
         $this->resolver->resolve($reflectionParameter);
     }
 
+    public function testCannotResolveReflectionParameterWithoutTypeOnFunction()
+    {
+        $this->expectException(UnsupportedException::class);
+        $this->expectExceptionMessage('Cannot resolve type for "fclose($stream)".');
+
+        $reflectionFunction = new \ReflectionFunction('fclose');
+        $reflectionParameter = $reflectionFunction->getParameters()[0];
```
Comment (OskarStark) [inline src/Symfony/Component/TypeInfo/Tests/TypeResolver/ReflectionParameterTypeResolverTest.php]:
```suggestion
        $reflectionFunction = new \ReflectionFunction('fclose');
        $reflectionParameter = $reflectionFunction->getParameters()[0];

        $this->expectException(UnsupportedException::class);
        $this->expectExceptionMessage('Cannot resolve type for "fclose($stream)".');        
```

File: src/Symfony/Component/TypeInfo/Tests/TypeResolver/ReflectionReturnTypeResolverTest.php [hunk 2]
```diff
@@ -44,6 +44,16 @@ public function testCannotResolveReflectionFunctionAbstractWithoutType()
         $this->resolver->resolve($reflectionFunction);
     }
 
+    public function testCannotResolveReflectionFunctionWithoutType()
+    {
+        $this->expectException(UnsupportedException::class);
+        $this->expectExceptionMessage('Cannot resolve type for "fopen()".');
+
+        $reflectionFunction = new \ReflectionFunction('fopen');
```
Comment (OskarStark) [inline src/Symfony/Component/TypeInfo/Tests/TypeResolver/ReflectionReturnTypeResolverTest.php]:
```suggestion
        $reflectionFunction = new \ReflectionFunction('fopen');
        
        $this->expectException(UnsupportedException::class);
        $this->expectExceptionMessage('Cannot resolve type for "fopen()".');        
```

Comment (alexandre-daubois) [reply src/Symfony/Component/TypeInfo/Tests/TypeResolver/ReflectionReturnTypeResolverTest.php]: Indeed 👍

**Review by OskarStark — APPROVED** [review]

**Review by Korbeil — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @alexandre-daubois.

**Pull request merged**